mock-express available for use

Hi team,

I would like to use [mock-express](https://github.com/howardabrams/node-mocks-http/tree/master/lib/express) to test out some init files in my Express.js application. These init files, among other functions, send my express app through app.use ([example](https://github.com/punchcard-cms/punchcard/blob/a30e5c2ecc8b9072d671365bcc802e9c5b03ef40/lib/init/logger.js).)

I don't see a way to access mock-express.js. I've tried things like `node-mocks-http/lib/express/mock-express`, but then that fails with errors like: 

`Error: Cannot find module './node/http'`

Could I create a PR to add mock-express into the `exports` at the bottom of `/lib/http-mock.js`?

```
var response = require('./mockResponse');
var express = require('./express/mock-express');

...

exports.createResponse = response.createResponse;
exports.createExpress = express;
exports.createMocks = createRequestResponse;
```

Have you tried this and would that simple change open up mock-express to be used, or would there be more changes that would be required to mock up my Express application?

thanks,
Scott

--- a/lib/express/mock-express.js
+++ b/lib/express/mock-express.js
@@ -6,7 +6,7 @@ var mixin = require('merge-descriptors');
 
 var application = require('./mock-application');
 var request = require('./mock-request');
-var response = require('./mock-response');
+var response = require('../mockResponse');
 
 function createApplication() {
   var app = function() {};
@@ -19,7 +19,7 @@ function createApplication() {
     app: app
   };
   app.response = {
-    __proto__: response,
+    __proto__: response.createResponse(),
     app: app
   };
   app.init();

--- a/lib/express/mock-request.js
+++ b/lib/express/mock-request.js
@@ -8,7 +8,7 @@ var isIP = require('net').isIP;
 var fresh = require('fresh');
 
 var defineGetter = require('./utils/define-getter');
-var http = require('./node/http');
+var http = require('http');
 
 var req = exports = module.exports = {
   __proto__: http.IncomingMessage.prototype

--- a/lib/http-mock.js
+++ b/lib/http-mock.js
@@ -9,6 +9,7 @@
 
 var request = require('./mockRequest');
 var response = require('./mockResponse');
+var express = require('./express/mock-express');
 
 /**
  * Creates linked req and res objects. Enables using methods that require both
@@ -30,3 +31,4 @@ var createRequestResponse = function (reqOpts, resOpts) {
 exports.createRequest = request.createRequest;
 exports.createResponse = response.createResponse;
 exports.createMocks = createRequestResponse;
+exports.express = express;

--- a/package.json
+++ b/package.json
@@ -41,8 +41,15 @@
   },
   "dependencies": {
     "accepts": "^1.3.3",
+    "depd": "^1.1.0",
+    "fresh": "^0.3.0",
     "lodash.assign": "^4.0.6",
+    "merge-descriptors": "^1.0.1",
+    "methods": "^1.1.2",
     "mime": "^1.3.4",
+    "net": "^1.0.2",
+    "parseurl": "^1.3.1",
+    "range-parser": "^1.2.0",
     "type-is": "^1.6.4"
   },
   "devDependencies": {

